Description: Link libm2s-glew, libm2s-glut, and libm2s-glu against libm2s-opengl
 libm2s-glew, libm2s-glut, and libm2s-glu require symbols from libm2s-opengl but
 are not linked against this. Fix this link error by adding a line to the
 respective Makefile.am.
 .
 multi2sim (4.1+svn20130603-1) unstable; urgency=low
 .
   * Link libm2s-glew, libm2s-glut, and libm2s-glu against libm2s-opengl
Author: William Panlener <wpanlener@gmail.com>

---

--- multi2sim-4.1+svn20130603.orig/runtime/glu/Makefile.am
+++ multi2sim-4.1+svn20130603/runtime/glu/Makefile.am
@@ -3,7 +3,7 @@ if HAVE_M32_FLAG
 lib_LTLIBRARIES = $(top_builddir)/lib/libm2s-glu.la
 
 __top_builddir__lib_libm2s_glu_la_LDFLAGS = -version-info 1:0:0
-
+__top_builddir__lib_libm2s_glu_la_LIBADD = -L$(top_builddir)/lib -lm2s-opengl
 __top_builddir__lib_libm2s_glu_la_SOURCES = \
 	\
 	api.c \
--- multi2sim-4.1+svn20130603.orig/runtime/glut/Makefile.am
+++ multi2sim-4.1+svn20130603/runtime/glut/Makefile.am
@@ -3,7 +3,7 @@ if HAVE_M32_FLAG
 lib_LTLIBRARIES = $(top_builddir)/lib/libm2s-glut.la
 
 __top_builddir__lib_libm2s_glut_la_LDFLAGS = -version-info 1:0:0
-
+__top_builddir__lib_libm2s_glut_la_LIBADD = -L$(top_builddir)/lib -lm2s-opengl
 __top_builddir__lib_libm2s_glut_la_SOURCES = \
 	\
 	api.c \
--- multi2sim-4.1+svn20130603.orig/runtime/glew/Makefile.am
+++ multi2sim-4.1+svn20130603/runtime/glew/Makefile.am
@@ -3,7 +3,7 @@ if HAVE_M32_FLAG
 lib_LTLIBRARIES = $(top_builddir)/lib/libm2s-glew.la
 
 __top_builddir__lib_libm2s_glew_la_LDFLAGS = -version-info 1:0:0
-
+__top_builddir__lib_libm2s_glew_la_LIBADD = -L$(top_builddir)/lib -lm2s-opengl
 __top_builddir__lib_libm2s_glew_la_SOURCES = \
 	\
 	api.c \
